eclipse-theia/theia
# repo: give proper `id` for builtin themes

#### What it does


The pull-request updates the root `package.json` to use the proper `id` for builtin themes so they are not re-downloaded by the `extension-pack`.

#### How to test


1. perform `yarn`
2. perform `yarn download:plugins`
3. confirm that themes are not installed twice under `plugins/` and you should see the following logs for each plugin when attempting to resolve the `extension-pack`:

    ```
    - vscode.theme-monokai: already downloaded - skipping
    ````


#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
